src/components/DataBrowser.vue

```diff
@@ -93,6 +93,9 @@ export default {
         this.$emit('input', val);
       },
     },
+    itemsPerPageInternal() {
+      return this.options.itemsPerPage;
+    },
```

Comment:
What is the point of this computed property?

`'options.itemsPerPage'` works as a watcher key.

https://vuejs.org/v2/api/#watch

Reply:
I knew that at one point and for some reason just forgot about it.  It's interesting what your brain does to work around stuff when you forget about specific pieces of information.